corehq/sql_accessors/sql_templates/get_closed_case_ids.sql

```diff
@@ -0,0 +1,13 @@
+DROP FUNCTION IF EXISTS get_closed_case_ids(text, text);
+
+CREATE FUNCTION get_closed_case_ids(domain_name text, p_owner_id text) RETURNS TABLE (case_id VARCHAR(255)) AS $$
+BEGIN
+    RETURN QUERY
+    SELECT case_table.case_id FROM form_processor_commcarecasesql AS case_table
+    WHERE
+      case_table.closed = TRUE
+      AND case_table.domain = domain_name
+      AND case_table.owner_id = p_owner_id
```

Comment:
Did this one need to check `modified_by` in case `owner_id` is blank? (Though no need to update if we can confirm that we'll set `owner_id` appropriately on initial migration)